packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -442,7 +454,7 @@ export class ServerlessCluster extends ServerlessClusterBase {
       databaseName: props.defaultDatabaseName,
       dbClusterIdentifier: clusterIdentifier,
       dbClusterParameterGroupName: clusterParameterGroupConfig?.parameterGroupName,
-      dbSubnetGroupName: this.subnetGroup.subnetGroupName,
+      dbSubnetGroupName: this.subnetGroup?.subnetGroupName,
```

Comment:
Is it actually legal to pass `dbSubnetGroupName` without `vpcSecurityGroupIds`? Seems kind of weird...

Reply:
I have no idea, the [CFN doc](https://docs.aws.amazon.com/fr_fr/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbcluster.html#aws-resource-rds-dbcluster-properties) treats them independently

Reply:
cf https://github.com/aws/aws-cdk/pull/17413/commits/66e9b55ae6f405a258d88a32a44480b3258c12d9#r781963251